ush/gfs_nceppost.sh

```diff
@@ -274,6 +274,7 @@ fi
 
 # allow threads to use threading in Jim's sp lib
 # but set default to 1 
+export OMP_NUM_THREADS=$threads
```

Comment:
@JesseMeng-NOAA This change might have impact on the config. of the upstream global workflow. Please remove it.

Reply:
Done!